lib/google/ads/google_ads/logging_interceptor.rb

```diff
@@ -16,13 +16,20 @@
 #
 # Interceptor to log outgoing requests and incoming responses.
 
+require 'google/ads/google_ads/v1/errors/errors_pb'
+require 'google/ads/google_ads/v0/errors/errors_pb'
 require 'grpc/generic/interceptors'
 require 'json'
 
 module Google
   module Ads
     module GoogleAds
       class LoggingInterceptor < GRPC::ClientInterceptor
+        INTERESTING_ERROR_CLASSES = [
```

Comment:
Can we generate this using the KNOWN_VERSIONS array in Google::Ads::GoogleAds instead, so it updates automatically when we modify that canonical list?

Reply:
done!